controllers/apps/transformer_cluster_backup_policy.go

```diff
@@ -733,3 +736,11 @@ func mergeSchedulePolicy(src *dpv1alpha1.SchedulePolicy, dst *dpv1alpha1.Schedul
 		dst.CronExpression = src.CronExpression
 	}
 }
+
+func addOneDay(retentionPeriod dpv1alpha1.RetentionPeriod) dpv1alpha1.RetentionPeriod {
+	duration, err := retentionPeriod.ToDuration()
+	if err != nil {
+		return ""
+	}
+	return dpv1alpha1.RetentionPeriod(fmt.Sprintf("%d", int(duration.Minutes()/1440)+1))
```

Comment:
A legal RetentionPeriod should be in "number + unit" format, e.g. "10d". Also, I suggest using the smallest unit "m" (minutes) instead of "d".